### PR TITLE
Divyanshu | Test | Provide setting in branch level to run the auto-paid automatically

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86cu262x3
https://app.clickup.com/t/86cu246ye

### CODE REVIEW

File: apps/web-giddh/src/app/invoice/settings/invoice.settings.component.html [hunk 1]
```diff
@@ -760,6 +760,31 @@ <h4 class="clearfix bd-rb font-16">{{ commonLocaleData?.app_invoice }} <span></s
                                         </div>
                                     </div>
                                 </div>
+                                <div class="clearfix pd-t1" *ngIf="voucherApiVersion === 2">
+                                    <div class="row">
+                                        <div class="col-sm-12 col-md-12 col-lg-3">
+                                            <mat-slide-toggle
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/settings/invoice.settings.component.html:766]: remove mobile specific code

File: apps/web-giddh/src/app/invoice/settings/invoice.settings.component.ts [hunk 2]
```diff
@@ -562,4 +565,29 @@ export class InvoiceSettingComponent implements OnInit, OnDestroy {
...
+                data: {
+                    title: this.commonLocaleData?.app_confirm,
+                    body: this.localeData?.auto_paid_confirmation_message,
+                    ok: this.commonLocaleData?.app_yes,
+                    cancel: this.commonLocaleData?.app_no
+                }
+            }).afterClosed()
+                .subscribe((response: boolean) => {
+                    dialogRef.unsubscribe();
+                    this.invoiceSetting.autoGenerateVoucherFromEntry = response ? true : false;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/settings/invoice.settings.component.ts]:
```suggestion
                    this.invoiceSetting.autoGenerateVoucherFromEntry = response;
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/invoice/settings/invoice.settings.component.html (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/invoice/settings/invoice.settings.component.html:766]: This used for 991px screen size

File: apps/web-giddh/src/app/invoice/settings/invoice.settings.component.ts (hunk 2 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/invoice/settings/invoice.settings.component.ts]: Done, Thanks

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**